raiden/tests/integration/cli/test_cli.py

```diff
@@ -114,13 +121,11 @@ def test_cli_wrong_rpc_endpoint(cli_args):
 def test_cli_wrong_network_id_try_mainnet(cli_args):
     child = spawn_raiden(cli_args)
     try:
-        child.expect('Welcome to Raiden')
-        child.expect('The following accounts were found in your machine:')
-        child.expect('Select one of them by index to continue: ')
-        child.sendline('0')
+        expect_cli_until_account_selection(child)
         child.expect(
             "The chosen ethereum network 'mainnet' differs from the ethereum "
-            "client 'smoketest'")
+            "client 'smoketest'"
```

Comment:
Lint complaining about the missing comma here

Reply:
ofcourse it does -.-

Reply:
I really don't get it why locally I am not getting these errors when I run flake8.